src/httpserver.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include <chainparamsbase.h>
 #include <compat.h>
+#include <netaddress.h> // For explicitness; already included via netbase.h
```

Comment:
no need for this inline comment

Reply:
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#source-code-organization

Reply:
@jnewbery agreed, I added it for reviewers so they wouldn't wonder why this was added ;)

@hebasto if you are referring to *Every .cpp and .h file should #include every header file it directly uses classes, functions or other definitions from, even if those headers are already included indirectly through other headers.*... yes, this is why I added the #include. Are you posting the link for a different reason?

Reply:
> I added it for reviewers so they wouldn't wonder why this was added

You can leave github comments on your own PR for that. If we added a code comment "// already included via .... " to all includes that were already included indirectly, then most of the includes would have that comment.

Reply:
Agreed

Reply:
> Are you posting the link for a different reason?

No, you point the reason exactly :)

Reply:
done